### PR TITLE
Fixes #33403 - Add Version Details View Table

### PR DESCRIPTION
**Pull request by Andrewgdewar**

### CODE REVIEW

File: webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailConfig.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+});
+
+
+export default ({ cvId, versionId }) => [
+  {
+    name: __('Components'),
+    getCountKey: item => item?.component_view_count,
+    responseSelector: state => selectCVVersions(state, cvId),
+    statusSelector: state => selectCVVersionsStatus(state, cvId),
+    autocompleteEndpoint: '',
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailConfig.js:67]: Think it's ok to use /katello/api/v2/content_view_versions/auto_complete_search?content_view_id=3 with the search enabled here for consistency and searching based on repository. 

Also, CV Name is appended with version which isn't required. 
Screenshot from 2021-09-17 13-47-15

File: webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      name: __('Ansible collections'),
+      url: `ansible_collections?repositoryId=${libraryInstanceId}`,
+    },
+    deb: {
+      name: __('Deb packages'),
+      url: `debs?repositoryId=${libraryInstanceId}`,
+    },
+    docker_manifest: {
+      name: __('Docker manifests'),
+    },
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js:28]: URL : products/1/repositories/72/content/content/docker_manifests?page=1&per_page=20

File: webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    docker_manifest_list: {
+      name: __('Docker manifest lists'),
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js:30]: 'Container manifest lists' for consistency on the page.
Also could you update it in the versions additional content component to use container over docker.
URL takes the form: products/1/repositories/72/content/content/docker_manifest_lists?page=1&per_page=20

**Review by sjha4 — COMMENTED** [review]
Minor comments..Looks great overall!

File: webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js (hunk 2 above)
Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js:28]: updated

File: webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js (hunk 3 above)
Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js:30]: Thank you! Addressed all of the above

File: webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailConfig.js (hunk 1 above)
Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailConfig.js:67]: I attempted this, and the components endpoint dies when you attempt to give it a search param.

I fixed the naming issue.

**Comment by sjha4** [comment]
Looks like this doesn't have the commit you need for the versions table and instead has the routing part for some reason.

**Comment by sjha4** [comment]
For some reason, when a task is publishing, the version link still redirects to the old page. 

Everything else looks good other than the unnecessary commits.

**Pull request closed without merging**